RecoEgamma/EgammaPhotonProducers/python/photons_cfi.py

```diff
@@ -80,7 +80,13 @@
     RecHitSeverityToBeExcludedEE = cleanedHybridSuperClusters.RecHitSeverityToBeExcluded,
 )
 
-photonsFromMultiCl = photons.clone(
-  photonProducer = 'photonCoreFromMultiCl'
-)
-
+photonsFromMultiCl = photons.clone()
+# Change underlying C++ type, as ugly as it could be, and add required
+# configuration parameters
```

Comment:
Given that photonsFromMultiCl does not have much to do with photons defined earlier, please revert photons_cfi.py to its original state in the release and make photonsFromMultiCl_cff.py with its definitions which could be simply a clone from ootPhotons with the change of the input collection and reco step.